diff --git a/conf.in.py b/conf.in.py
index 6515474..b550ab5 100644
--- a/conf.in.py
+++ b/conf.in.py
@@ -1,7 +1,6 @@
 import os
 import sys
 
-import sphinx_rtd_theme
 import yaml
 
 # If extensions (or modules to document with autodoc) are in another directory,
@@ -73,8 +72,7 @@ pygments_style = "sphinx"
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]  # use with module import
+html_theme = 'python_docs_theme'
 
 # html_theme_path = ['.']  # use with git submodule
 
@@ -89,6 +87,7 @@ html_theme_options = {
     "titles_only": True,
     "versioning": True,
     "canonical_url": "https://qgis.org/pyqgis/latest",
+    "nosidebar": True
 }
 
 # Add any paths that contain custom themes here, relative to this directory.
