diff --git a/conf.in.py b/conf.in.py
index ee71be3..6d2588b 100644
--- a/conf.in.py
+++ b/conf.in.py
@@ -2,7 +2,6 @@ import os
 import sys
 from pathlib import Path
 
-import sphinx_rtd_theme
 import yaml
 
 # If extensions (or modules to document with autodoc) are in another directory,
@@ -76,8 +75,7 @@ pygments_style = "sphinx"
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]  # use with module import
+html_theme = 'python_docs_theme'
 
 # html_theme_path = ['.']  # use with git submodule
 
@@ -93,6 +91,7 @@ html_theme_options = {
     "versioning": True,
     "canonical_url": "https://qgis.org/pyqgis/latest",
     "navigation_depth": 1,
+    "nosidebar": True
 }
 
 # Add any paths that contain custom themes here, relative to this directory.
