diff --git a/conf.py.in b/conf.py.in
index 486f489..3239f58 100644
--- a/conf.py.in
+++ b/conf.py.in
@@ -3,8 +3,6 @@
 import sys, os
 import yaml
 
-import sphinx_rtd_theme
-
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
@@ -65,8 +63,7 @@ pygments_style = 'sphinx'
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'sphinx_rtd_theme'
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]  # use with module import
+html_theme = 'python_docs_theme'
 
 # html_theme_path = ['.']  # use with git submodule
 
@@ -81,6 +78,7 @@ html_theme_options = {
   'titles_only': True,
   'versioning': True,
   'canonical_url': 'https://qgis.org/pyqgis/latest',
+  'nosidebar': True
 }
 
 # Add any paths that contain custom themes here, relative to this directory.
